enginetest/queries/script_queries.go

```diff
@@ -2273,6 +2273,55 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "having clause without groupby clause, all rows implicitly form a single aggregate group",
+		SetUpScript: []string{
+			"create table numbers (val int);",
+			"insert into numbers values (1), (2), (3);",
+			"insert into numbers values (2), (4);",
+		},
+		Assertions: []ScriptTestAssertion{
```

Comment:
While you're in here... could you see if the related skipped tests in `column_alias_queries.go` that try to use having and hit this same error will work now and unskip them if so?